docs/index.html

```diff
@@ -1,96 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This is bucklescript.github.io/bucklescript. Not needed anymore

Reply:
do you need a redirect?

Reply:
Nope. This page doesn't contain much info. We can let it 404 and folks can just point to the new site. We have the redirect logic for all other pages, but not turned on yet.